examples/gaussian_processes/gaussian_process.ipynb

```diff
@@ -4,18 +4,23 @@
    "cell_type": "markdown",
```

Comment:
<p>I would annotate the dimensions of <code>mu_post</code> and <code>sigma_post</code>. It could even be something as <code>"pred_id"</code> for mu and <code>("pred_id", "pred_id2")</code> for sigma (or pred_id_bis). Or anything really.</p><p> </p><p>I would also move the definition of xgrid, D_pred and D_off_diag to be outside the model context. They are all numpy arrays "only" which I don't think is very clear with the current code structure.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/404/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>In that case, should we remove also the comments "Posterior mean" and "Posterior covariance"?</p><p> </p>

Reply:
<p>I think the comments are ok</p>